### ISSUE TITLE
DARK MODE ENHANCEMENT

### ISSUE THREAD

**Issue by Dev-i-l** [brave/brave-browser#11853]
ADD THE OPTION TO TOGGLE DARK MODE IN THREE DOT MENU

**Comment by srirambv** [comment]
Dupe of #11085

**Comment by Dev-i-l** [comment]
AS OF NOW WE CAN ENABLE DARK MODE ONLY BY GOING INTO SETTINGS, SO I SUGGEST ADDING THE OPTION IN THREE DOT MENU FOR EASY ACCESSIBILITY, HENCE MY ISSUE IS NOT THE SAME AS #11085

**Comment by Dev-i-l** [comment]
> Dupe of #11085

AS OF NOW WE CAN ENABLE DARK MODE ONLY BY GOING INTO SETTINGS, SO I SUGGEST ADDING THE OPTION IN THREE DOT MENU FOR EASY ACCESSIBILITY, HENCE MY ISSUE IS NOT THE SAME AS #11085 AND IT SHOULD BE REOPENED
